c_emulator/riscv_platform.c

```diff
@@ -82,6 +82,15 @@ uint64_t sys_pmp_grain(unit u)
   return rv_pmp_grain;
 }
 
+uint64_t sys_vector_vlen_exp(unit)
+{
+  return rv_vector_vlen_exp;
+}
+uint64_t sys_vector_elen_exp(unit)
```

Comment:
Slightly ocd (we should get `clang-format`) but can you add a blank line between these functions?